### PR TITLE
Fixed supporting for Django 2.0 and greater

### PR DESCRIPTION
**Pull request by koddr**
* Add fix for supporting Django `2.0` and greater
* Add fallback for Django `1.11` and older

### CODE REVIEW

File: django_rq/urls.py [hunk 1]
```diff
@@ -1,34 +1,73 @@
...
+        re_path(r'^queues/(?P<queue_index>[\d]+)/(?P<job_id>[-\w]+)/delete/$',
+                views.delete_job, name='rq_delete_job'),
+        re_path(r'^queues/actions/(?P<queue_index>[\d]+)/$',
+                views.actions, name='rq_actions'),
+        re_path(r'^queues/(?P<queue_index>[\d]+)/(?P<job_id>[-\w]+)/requeue/$',
+                views.requeue_job_view, name='rq_requeue_job'),
+    ]
+
+# Fallback for Django 1.11 and lower
+if django.VERSION[0] == 1:
```
Comment (selwin) [inline django_rq/urls.py]: We should avoid declaring all the URL paths twice. According to the docs: `The django.conf.urls.url() function from previous versions is now available as django.urls.re_path()`. So you should be able to do this:
```python
try:
    from django.urls import re_path  # Django 2.x
except ImportError:
    from django.conf.urls import url as re_path # Django 1.x

urlpatterns = [
    re_path(r'^$', views.stats, name='rq_home'),
    ...
]


```

Comment (koddr) [reply django_rq/urls.py]: Thanks, I fixed it.

**Comment by selwin** [comment]
Thanks!

**Comment by koddr** [comment]
@selwin waiting for new version!

**Comment by selwin** [comment]
I'll make a new release this weekend or early next week. Hoping this https://github.com/rq/django-rq/pull/276 is ready by then.

**Pull request merged**